tfhe/src/integer/gpu/ciphertext/compressed_ciphertext_list.rs

```diff
@@ -173,43 +173,12 @@ impl CudaCompressedCiphertextList {
     ///     .push(d_ct3, &streams)
     ///     .build(&cuda_compression_key, &streams);
     ///
-    /// let reference_compressed = CompressedCiphertextListBuilder::new()
-    ///     .push(ct1)
-    ///     .push(ct2)
-    ///     .push(ct3)
-    ///     .build(&compression_key);
-    ///
```

Comment:
you want to have dedicated tests down the line ?

Reply:
(was thinking about a should run that could trigger if any CPU/GPU code gets changed if we need consistency/determinism between CPU and GPU later)

Reply:
We can't ensure determinism between CPU & GPU for decompression since it involves the FFT actually.

Reply:
it's true, but isn't only the compression tested ?

Reply:
no it was the decompression

Reply:
oh sorry I removed the compression by mistake

Reply:
Let me fix it

Reply:
ah indeed, funny it worked, though I think Sam said as much in some context it can work

Reply:
Hmm it's failing though, but the packing ks decomposition looks correct.

Reply:
oh my... plus it's pure integer ops during compression, it should be deterministic